Content.IntegrationTests/SetupCompileDM.cs

```diff
@@ -7,19 +7,29 @@
 // ReSharper disable once CheckNamespace
 public sealed class SetupCompileDm {
     public const string TestProject = "DMProject";
-    public const string Environment = "environment.dme";
+    public const string TestDirectory = "Tests";
+    public const string BaseEnvironment = "environment.dme";
+    public const string Environment = "test_environment.dme";
 
     public static readonly string BaseDirectory = AppDomain.CurrentDomain.BaseDirectory;
+    public static readonly string BaseDmEnvironment = Path.Join(BaseDirectory, TestProject, BaseEnvironment);
     public static readonly string DmEnvironment = Path.Join(BaseDirectory, TestProject, Environment);
+    public static readonly string TestFilesDirectory = Path.Join(BaseDirectory, TestProject, TestDirectory);
     public static readonly string CompiledProject = Path.ChangeExtension(DmEnvironment, "json");
 
     [OneTimeSetUp]
     public void Compile() {
         DMCompiler.DMCompiler compiler = new();
+        // open the DME and append include statements for each test
```

Comment:
You could include the files in the `Files` list a few lines down instead

Reply:
I always forget you can do that